janus_core/calculations/geom_opt.py

```diff
@@ -85,6 +87,8 @@ class GeomOpt(BaseCalculation):
     write_kwargs
         Keyword arguments to pass to ase.io.write to save optimized structure.
         Default is {}.
+    write_trajectory
```

Comment:
I'm open to either way, but maybe `write_traj`, since we have `traj_kwargs` etc.?